Python/5_smallest_multiple.py

```diff
@@ -1,13 +1,11 @@
 # Problem 5
-def multiplies(n):
-	a = [i for i in xrange(11,21) if n%i==0]
+def get_multiplies(n, multiplies):
```

Comment:
In this solution you will call this function very many times. And for each call new array xrange(1, multiplies + 1) will be defined. 

I think it's better to define it once because it's not changed through program lifecycle